src/oauth.c

```diff
@@ -146,7 +151,22 @@ apr_byte_t oidc_oauth_get_bearer_token(request_rec *r,
 				/* copy the result in to the access_token */
 				*access_token = apr_pstrdup(r->pool, auth_line);
 
-			} else {
+				known_scheme = true;
+
+			} else if (accept_token_in & OIDC_OAUTH_ACCEPT_TOKEN_IN_BASIC) {
+
+				char *decoded_line = ap_pbase64decode(r->pool, auth_line);
```

Comment:
it appears that `ap_pbase64decode` is deprecated in favor of `apr_pbase64_decode `

Reply:
Right, though that's in the Apache httpd/APR trunk, and not in any released APR version as of yet:

````
[puiterwijk@workstation apr (trunk u=)]$ git branch -r --contains e4ceec2062
  origin/HEAD -> origin/trunk
  origin/trunk
````

Reply:
I've decided to just add `oidc_parse_base64` from `parse.c` to the header and use that instead.